articles/hdinsight/interactive-query/apache-hive-migrate-workloads.md

```diff
@@ -52,7 +52,7 @@ This step uses the [`Hive Schema Tool`](https://cwiki.apache.org/confluence/disp
 
 1. Create a temporary HDInsight 4.0 cluster to access the 4.0 Hive `schematool`. You can use the [default Hive metastore](../hdinsight-use-external-metadata-stores.md#default-metastore) for this step.
 
-1. From the HDInsight 4.0 cluster, execute `schematool` to upgrade the target HDInsight 3.6 metastore:
+1. From the HDInsight 4.0 cluster, execute `schematool` to upgrade the target HDInsight 3.6 metastore. Open an ssh session on headnode and follow the steps below:
```

Comment:
Since this is the first mention of SSH to the HDInsight headnode, it would be better to put a relative link to this article: https://docs.microsoft.com/azure/hdinsight/hdinsight-hadoop-linux-use-ssh-unix.

And more precise to change "follow the steps below" to "run the following script".